src/index.js

```diff
@@ -1,8 +1,10 @@
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
!comment

Reply:
outdated

Reply:
that is not a comment, instead is a type of special syntax that informs `eslint` to disable a default behaviour.

Reply:
Great